Core/test/qa-functional/src/org/sleuthkit/autopsy/commonpropertiessearch/InterCaseTestUtils.java

```diff
@@ -204,7 +202,6 @@
         //kitchen sink
         ArrayList<IngestModuleTemplate> kitchenSink = new ArrayList<>();
         kitchenSink.add(exifTemplate);
-        kitchenSink.add(iOsTemplate);
```

Comment:
*Unrelated* to your changes, but since I pulled down most recent develop (to ignore device accounts) I had to change line 223 in this file:

I had to change the call to getCorrelationTypes() to getDefinedCorrelationTypes(). 

You may want to make that change in your PR before merging it.